pillar/vault/roles/pki.sls

```diff
@@ -0,0 +1,46 @@
+{% set env_settings = salt.cp.get_file_str("salt://environment_settings.yml")|load_yaml %}
+{% set ttl = '8760h' %}
```

Comment:
Rename this variable to be a descriptive constant (e.g. `THIRTY_DAYS, or whatever it actually is)

Reply:
Doesn't that defeat the purpose since then if/when you want to change the variable you'd have to change it everywhere you've used it?